packages/custom-decorators/package.json

```diff
@@ -36,7 +36,8 @@
     "url": "https://github.com/algoan/nestjs-components/issues"
   },
   "dependencies": {
-    "jwt-decode": "^4.0.0"
+    "jwt-decode": "^4.0.0",
+    "class-validator": "^0.14.0"
```

Comment:
Maybe this should be a peer dependency instead?

Reply:
Ah! I didn't know that. Then, I think it would be better to create a new package `@algoan/nestjs-class-validator` instead of updating `@algoan/nestjs-custom-decorators`.

The current decorators in this package are dedicated to parameters instead of validation and are few in numbers. Adding the whole class-validator will create some confusion, I fear. Hence, a new package.